Add i4p and Trident HSM (#3)
pkic/pqccm: README.md

OQS is an open source software library that implements PQC [algorithms](https://openquantumsafe.org/liboqs/algorithms/), as well as integrations such as into OpenSSL.

## I4P
Trident HSM is a general purpose network HSM that combines the high-level hardware security and multi-party computation to provide the highest level of data protection required in business.
It received the Common Criteria (CC) EAL4+ (EN 419221-5 as well as EN 419241-2) certification which is the highest level of certification available for HSM modules in Europe.

> 

Reviewer: Please focus on the PQ functionality of the product

Author: ```suggestion
Trident HSM is a general purpose network HSM that received the Common Criteria (CC) EAL4+ (EN 419221-5 as well as EN 419241-2) certification. Cryptographic applications are enabled to use Post-Quantum algorithms even in the CC restricted mode that helps with using hybrid solutions and the transition to PQC.

```

Reviewer: Can you clarify if the Post-Quantum algorithms are covered by the CC certification? If so, under what standardization/version etc?

Author: ```suggestion
Trident HSM is a general purpose network HSM that combines the high-level hardware security and multi-party computation to provide the highest level of data protection required in business.
It received the Common Criteria (CC) EAL4+ (EN 419221-5 as well as EN 419241-2) certification which is the highest level of certification available for HSM modules in Europe.

SPHINCS+ and Kyber algorithms are covered by CC certification for different Trident versions according to standards and implementation versions:

Trident v2.1.3:
* SPHINCS+ "Daniel J. Bernstein, Andreas Hülsing, Stefan Kölbl: The SPHINCS+ Signature Framework", September 23, 2019, https://sphincs.org/data/sphincs+-paper.pdf

Trident v3.1 (CC certification is expected by Q1 2023):

- **SPHINCS+ v.3.1** "Jean-Philippe Aumasson, Daniel J. Bernstein, Ward Beullens, Christoph Dobraunig, Maria Eichlseder, Scott Fluhrer, Stefan-Lukas Gazdag, Andreas Hülsing, Panos Kampanakis, Stefan Kölbl, Tanja Lange, Martin M. Lauridsen, Florian Mendel, Ruben Niederhagen, Christian Rechberger, Joost Rijneveld, Peter Schwabe, Bas Westerbaan: SPHINCS+ v.3.1.", June 10, 2022, https://sphincs.org/data/sphincs+-r3.1-specification.pdf"
- **Kyber** "Roberto Avanzi, Joppe Bos, Léo Ducas, Eike Kiltz, Tancrède Lepoint, Vadim Lyubashevsky, John M. Schanck, Peter Schwabe, Gregor Seiler, Damien Stehlé: CRYSTALS-Kyber, Algorithm Specifications And Supporting Documentation (version 3.02), August 4, 2021, https://pqcrystals.org/kyber/data/kyber-specification-round3-20210804.pdf"
- **NTRU** "Cong Chen, Oussama Danba, Jeffrey Hoffstein, Andreas Hulsing, Joost Rijneveld, John M. Schanck, Peter Schwabe, William Whyte, Zhenfei Zhang, Tsunekazu Saito, Takashi Yamakawa, Keita Xagawa: NTRU, Algorithm Specifications And Supporting Documentation, September 30, 2020, https://csrc.nist.gov/CSRC/media/Projects/post-quantum-cryptography/documents/round-3/submissions/NTRU-Round3.zip

```

> The certified product is capable of post-quantum algorithms for signature creation and key encapsulation mechanisms. Supported algorithms are SPHINCS+,Kyber and NTRU.

Author: ```suggestion
```

> See more [Link](https://www.i4p.com/)

Reviewer: Do you have a direct link, making references or providing documentation to the PQ implementation?

Author: No direct link unfortunately
